tartufo/cli.py

```diff
@@ -81,6 +81,13 @@ def get_command(self, ctx: click.Context, cmd_name: str) -> Optional[click.Comma
     show_default=True,
     help="Enable high signal regexes checks.",
 )
+@click.option(
+    "--scan-filenames/--no-scan-filenames",
+    is_flag=True,
+    default=True,
+    show_default="--scan-filenames",
```

Comment:
```suggestion
    show_default=True,
```
Turns out, setting this to a string does not work as expected. But with `click >= 8.0`, these will display the string rather than the bool.